src/Containers/PictureCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { MainInterface, ResponsiveInterface } from '@Utils/BaseStyles';
+import styled from 'styled-components';
+import { StyledIcon } from 'styled-icons/types';
+import { flex, position } from '@Utils/Mixins';
+import { Card as C } from './Card';
+
+const Card = styled(C)`
+    ${'max-width:300px;'}
+`;
+
+interface PictureCardProps extends MainInterface, ResponsiveInterface {
+    image: string;
+    tags: { icon: StyledIcon; text: string }[];
+}
+
+export const PictureCard: React.FC<PictureCardProps> = ({
+    children,
+    image,
+    tags,
+    ...cardProps
+}): React.ReactElement => {
+    return (
+        <Card {...cardProps}>
+            <ImageWrapper>
+                <Image src={image} />
+                {tags.length ? (

```

Comment:
You can just do tags && (<Tags>....</Tags>)

Reply:
done